compiler/src/iree/compiler/Codegen/Interfaces/PartitionableLoopsInterface.cpp

```diff
@@ -139,6 +139,30 @@ struct NoPartitionableLoops : public PartitionableLoopsInterface::ExternalModel<
   }
 };
 
+struct ConcatOpPartitionableLoops
+    : public PartitionableLoopsInterface::ExternalModel<
+          ConcatOpPartitionableLoops, tensor::ConcatOp> {
+  llvm::SmallVector<unsigned>
+  getPartitionableLoops(Operation *op,
+                        std::optional<unsigned> maxNumPartitionedLoops) const {
+    auto concatOp = cast<tensor::ConcatOp>(op);
+    int64_t rank = concatOp.getResultType().getRank();
+    uint64_t concatDim = concatOp.getDim();
+    SmallVector<unsigned> partitionableLoops;
+    partitionableLoops.reserve(rank - 1);
+    for (unsigned i = 0; i < rank; ++i) {
+      if (i != concatDim) {
```

Comment:
Some comment as to why the concat dim is not part of the partitionable loops would help. Took me a while to come to the answer.